kikuchipy/signals/ebsd_master_pattern.py

```diff
@@ -270,6 +277,117 @@ def set_phase_parameters(
         phase = {k: v for k, v in inputs.items() if v is not None}
         _update_phase_info(self.metadata, phase, number)
 
+    def get_patterns(
+        self,
+        rotations: Rotation,
+        detector: EBSDDetector,
+        energy: int,
+        chunk_size: int,
+        dtype_out=np.float32,
+    ) -> LazyEBSD:
+        """
+        Creates a catalogue of EBSD patterns given a set of rotations and
+        a detector model from the master pattern.
+
+        Parameters
+        ----------
+        rotations : Rotation
+            Set of rotations to get patterns from.
+        detector : EBSDDetector
```

Comment:
There should be a comment here about whether many PCs or just one is allowed, and a check in the code, right?